var/spack/repos/builtin/packages/py-wxpython/package.py

```diff
@@ -11,13 +11,15 @@ class PyWxpython(PythonPackage):
     homepage = "https://www.wxpython.org/"
     pypi = "wxPython/wxPython-4.0.6.tar.gz"
 
+    version("4.2.2", sha256="5dbcb0650f67fdc2c5965795a255ffaa3d7b09fb149aa8da2d0d9aa44e38e2ba")
     version("4.1.1", sha256="00e5e3180ac7f2852f342ad341d57c44e7e4326de0b550b9a5c4a8361b6c3528")
     version("4.0.6", sha256="35cc8ae9dd5246e2c9861bb796026bbcb9fb083e4d49650f776622171ecdab37")
 
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
 
     depends_on("wxwidgets +gui")
+    depends_on("wxwidgets@3.2.6 +gui", when="@4.2.2")
 
     # Needed for the build.py script
     depends_on("py-setuptools", type="build")
```

Comment:
While we're applying edits here, I have locally:
```suggestion
    depends_on("py-setuptools", type="build")
    depends_on("py-setuptools@:75", type="build", when="@:4.1")  # deprecated license-file
```
since the next setuptools version will fail to parse the deprecated `license-file` (dashes) field in setup.cfg (ref: https://github.com/wxWidgets/Phoenix/commit/87b606aa041eba8274d6ca50da015f92070d74c6).